src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,124 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+
+    private static final int DEFAULT_ITEMS_NUMBER = 10;
```

Comment:
```suggestion
    private static final int DEFAULT_ITEMS_NUMBER = 10;
```
no empty line after class definition is needed per mate code style, other projects may have different conventions